src/styles/block/card.scss

```diff
@@ -0,0 +1,114 @@
+$fontColorBlue: #060b35;
+$fontColorGray: #616070;
+$buttonColor: #00acdc;
+$cardBorderColor: #f3f3f3;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  align-items: center;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid $cardBorderColor;
+
+  background: #fff;
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+
+    margin: 0 0 40px;
+
+    background-image: url(../images/imac.jpeg);
```

Comment:
You already have this image in `img` tag. Why did you put it to the background-image too?